.circleci/config.yml

```diff
@@ -74,7 +74,7 @@ jobs:
 
       - run: |
           set -euo pipefail
-          kurtosis run ./kurtosis/main.star --enclave ci-check "{\"kurtosis_package_indexer_version\": \"dev\", \"github_user_token\": \"${KURTOSISBOT_GITHUB_TOKEN}\"}"
+          kurtosis run ./kurtosis/main.star --enclave ci-check "{\"kurtosis_package_indexer_version\": \"dev\", \"github_user_token\": \"${KURTOSISBOT_GITHUB_TOKEN}\", \"snowflake_env\": {\"kurtosis_snowflake_account_identifier\": \"${KURTOSIS_SNOWFLAKE_ACCOUNT_IDENTIFIER}\", \"kurtosis_snowflake_db\": \"${KURTOSIS_SNOWFLAKE_DB}\", \"kurtosis_snowflake_password\": \"${KURTOSIS_SNOWFLAKE_PASSWORD}\", \"kurtosis_snowflake_role\": \"${KURTOSIS_SNOWFLAKE_ROLE}\", \"kurtosis_snowflake_user\": \"${KURTOSIS_SNOWFLAKE_USER}\", \"kurtosis_snowflake_warehouse\": \"${KURTOSIS_SNOWFLAKE_WAREHOUSE}\"}}"
```

Comment:
@leoporoli will we be hitting snowflake on every CI build? If so, will this be expensive? If so, is there a way to stub this out? If none of these, then all good.

Reply:
yes, it will hit Snowflake, but it should be cheap, I thought about this and because it will only be one query, and it can be compared with one of us doing a query in SF to check the package's run metrics.
But, yes probably worth it to avoid this just for the long term, so I've added a new `doNothingMetricsReporter` to avoid hitting Snowflake in the job.
Good call